oci/dependencies.bzl

```diff
@@ -33,3 +35,25 @@ def rules_oci_dependencies():
         strip_prefix = "bazel_features-1.10.0",
         url = "https://github.com/bazel-contrib/bazel_features/releases/download/v1.10.0/bazel_features-v1.10.0.tar.gz",
     )
+
+    http_archive(
+        name = "tar.bzl",
+        sha256 = "29a3c99c28deca5f8245e2fc32ffdb99c1ea69316462718f3bebfff441d36e4a",
+        strip_prefix = "tar.bzl-0.5.6",
+        url = "https://github.com/bazel-contrib/tar.bzl/releases/download/v0.5.6/tar.bzl-v0.5.6.tar.gz",
+    )
+
+    # Required bazel-lib dependencies
+
+    aspect_bazel_lib_dependencies()
+
+    # Register bazel-lib toolchains
+
+    aspect_bazel_lib_register_toolchains()
+
+    # Create the host platform repository transitively required by bazel-lib
+
+    maybe(
```

Comment:
Hmm, why this is needed? i thought we simply dropped this requirement with some release of bazel_lib.